app/frontend/components/pages/txHistory/transactionHistory.tsx

```diff
@@ -102,6 +106,10 @@ const MultiAsset = ({star, name, hash, amount}: MultiAssetProps) => (
       <span className="ellipsis">{hash.slice(0, -6)}</span>
       {hash.slice(-6)}
     </div>
+    <div className="multi-asset-hash">
+      <span className="ellipsis">{fingerprint.slice(0, -6)}</span>
+      {fingerprint.slice(-6)}
+    </div>
```

Comment:
These ellipses and slicing are getting repetitive, what would you say to a helper returning 
```
<span className="ellipsis">{whatever.slice(0, -6)}</span>{whatever.slice(-6)}
```
?

Reply:
Agree :+1: